recipes/default.rb

```diff
@@ -18,6 +18,8 @@
 # limitations under the License.
 #
 
+node.set['java']['install_flavor'] = 'windows' if platform_family?('windows')
```

Comment:
I think this is OK, in version 2 we'll refactor this to be less cryptic. Can we add an INFO message here though to let the user know we've changed the node value on them?